test/functional/services/common/browser.ts

```diff
@@ -298,11 +299,13 @@ export async function BrowserProvider({ getService }: FtrProviderContext) {
             dispatchEvent(target, dropEvent, dragStartEvent.dataTransfer);
             const dragEndEvent = createEvent('dragend');
             dispatchEvent(origin, dragEndEvent, dropEvent.dataTransfer);
-          }, 50);
+          }, 100);
       `,
         from,
         to
       );
+      // wait for 150ms to make sure the script has run
+      await delay(150);
```

Comment:
Not using PageObjects.common.sleep?

Reply:
I wanted to avoid introducing this dependency as the browser service is very basic (not even sure whether it work because the common page object relies on the browser service itself).